src/usercss/user.css

```diff
@@ -0,0 +1,92 @@
+.customform h1 {
```

Comment:
if this css is suppose to be for user component then we should have a User component file and that should import this css. Currently you are importing this css at App.js file which should not be dont.